README.md

```diff
@@ -200,3 +200,62 @@ $ docker build --target production --tag todo-app:prod .
 ```bash
 $ docker run -p 3000:3000 --env-file .env todo-app:prod
 ```
+
+# Test Container
+
+The test container is used to run the unit and integration tests for the To Do App. To build the test container, run the following command:
+
+```bash
+$ docker build --target test --tag todo-app:test .
+```
+
+```bash
+$ run: docker run --env-file ./.env.test todo-app:test
```

Comment:
Ah, here are the commands I asked for on the other review - never mind! 😄

Reply:
sorry I accidentally posted M8 in M9 and there was no way to undo it. M9 is still in progress